Library/Contributions/brew_bash_completion.sh

```diff
@@ -600,7 +600,10 @@ _brew ()
     done
 
     if [[ $i -eq $COMP_CWORD ]]; then
-        __brewcomp "$(brew commands --quiet --include-aliases)"
+        # Do not auto-complete "instal" abbreviation for "install" command.
+        # Prefix newline to prevent not checking the first command.
+        local cmds="\n$(brew commands --quiet --include-aliases)"
+        __brewcomp "${cmds//$'\n'instal$'\n'/$'\n'}"
```

Comment:
This is really just a minor detail and won't be noticeable performance-wise, but there should be no need to replace all instances in the string, because there should be only one line containing `instal`. To replace only one match:

```
"${cmds/$'\n'instal$'\n'/$'\n'}"
```